app/assets/stylesheets/framework/fonts.scss

```diff
@@ -0,0 +1,20 @@
+@font-face {
```

Comment:
I don't quite get why you need new fonts only for the registration page. I would think that the font we use on the rest of the pages also would work on a new registration page. If we want to change the font, we should change it everywhere, but again, this doesn't feel right as a drive-by with a new registration-page.

Reply:
Yeah as I said I want to rework the design but I thought about going page per page, but maybe it shouldn't be part of that PR.